src/main_controller.ts

```diff
@@ -75,26 +77,33 @@ export class MainController implements vscode.Disposable {
         const neovimPreScriptPath = path.posix.join(extensionPath, "vim", "vscode-neovim.vim");
         const neovimPostScriptPath = path.posix.join(extensionPath, "runtime/lua", "vscode-neovim/force-options.lua");
 
-        const args = [
+        const args = [];
+
+        if (config.useWsl) {
+            args.push("C:\\Windows\\system32\\wsl.exe");
```

Comment:
Windows handles `/` just fine, no need for all this escaping everywhere.

    args.push("C:/Windows/system32/wsl.exe");